llvm/lib/Target/X86/MCTargetDesc/X86BaseInfo.h

```diff
@@ -966,9 +966,12 @@ namespace X86II {
     NoTrackShift = EVEX_RCShift + 1,
     NOTRACK = 1ULL << NoTrackShift,
 
-    // Force VEX encoding
-    ExplicitVEXShift = NoTrackShift + 1,
-    ExplicitVEXPrefix = 1ULL << ExplicitVEXShift
+    // Force REX2/VEX/EVEX encoding
+    ExplicitOpPrefixShift = NoTrackShift + 1,
+    ExplicitREX2Prefix = 1ULL << ExplicitOpPrefixShift,
+    ExplicitVEXPrefix = 2ULL << ExplicitOpPrefixShift,
+    ExplicitEVEXPrefix = 3ULL << ExplicitOpPrefixShift,
```

Comment:
No use so far?

Reply:
Yes. As I said in the description. "ExplicitEVEXPrefix will be used by the instructions promoted to EVEX space for EGPR."

Reply:
It seems saying in description is less efficient than saying around the code :)

Reply:
Sure, let me add it in the code.

Reply:
Done